### ISSUE TITLE
arranging all files in proper folder and displaying all programs in a better way

### ISSUE THREAD

**Issue by Arisha2902** [Sagar0-0/DsA#711]
as we can all the codes are displayed randomly,one has to go through whole code to go to their desired code.

I will arrange codes in separate folder ,classifying them based on que topic and language

**Comment by Arisha2902** [comment]
@Sagar0-0 , I will help you mange your code files well, I don't have much experience but will try to give my best

**Comment by Sagar0-0** [comment]
Approved, please proceed ahead. @Arisha2902

**Comment by Arisha2902** [comment]
@Sagar0-0, 
Here I am arranging files in 3 major folders
![Screenshot (252)](https://github.com/user-attachments/assets/e98b7d62-913e-468c-b4f5-7f5311822f40)
![Screenshot (253)](https://github.com/user-attachments/assets/bcdd48f7-bdb2-4cfb-9d3f-e7b298a7fb6a)


Since the task is big , so do tell if this way is ok for you or should I change the way.

As well as you have not "assigned", me this issue rather than you have just approved.
 
Since the files are in large number, this task will take time

**Comment by Sagar0-0** [comment]
What folder structure do you suggest?
There are different approaches:
1. Problem type, array, linkedlist, etc.
2. Language wise(cpp, java)
3.  Platform wise(gfg,leetcode).

**Comment by Arisha2902** [comment]
i would like to suggest the first option i.e, "problem type" ,since people mosttly not check ques according to website and as for languge one i guess.

This can be future work to do ,since some que have solution both in java and cpp.

**Comment by Sagar0-0** [comment]
I agree, first one is better, but some problem can be or two types, example: linkedlist and recursion, or tree and recursion. How you are planing to handle that?

For one problem folder you can put thejava and cpp file there, this is already present for some problems.

**Comment by Arisha2902** [comment]
that is one of major issue for me also , as for thses ques i would focus on data Structure , if the question will only have recursion then it will be in recurssion only .
Otherwise it will seprated as per the Data Structure Used in Question.

**Comment by Sagar0-0** [comment]
Ok cool, go ahead, you can raise a PR when done

**Comment by Arisha2902** [comment]
@Sagar0-0 ,sir this project is quiet time taking and I have opened this  issue during "hacktoberfest" ,  to gain some points.
I have my test going on these days and doing this issue is quiet time taking ,so I will not be able do this.
